test/data/mock.php

```diff
@@ -216,6 +216,19 @@ protected function cspClean( $req ) {
 		unlink( $this->cspFile );
 	}
 
+	protected function errorWithScript( $req ) {
+		header( 'HTTP/1.0 404 Not Found' );
+        if ( isset( $req->query['scriptReply'] ) ) {
```

Comment:
I'd change the name from `scriptReply` to `withScriptContentType`; otherwise it looks a bit too similar to the `errorWithScript` name.